annotations-impl/src/main/java/org/ocpsoft/rewrite/annotation/scan/WebClassesFinder.java

```diff
@@ -174,13 +174,13 @@ private void handleClassEntry(URL entryUrl, String entryName, ClassVisitor visit
              */
             try
             {
-               classFileStream = entryUrl.openStream();
+              classFileStream = servletContext.getResourceAsStream(entryName);
             }
-            catch (IOException e)
+            catch (Exception e)
```

Comment:
The control flow of the code is exactly like before. But I agree, classFileStream could be null. I also would remove one of the try-catch-blocks. If you want me to change, I could. Possibly, it's more safe to keep the code like it was.